tests/AutoDocControllerTest.php

```diff
@@ -12,18 +12,24 @@ class AutoDocControllerTest extends TestCase
     use PHPMock;
 
     protected static array $documentation;
-    protected static string $localDriverFilePath;
+    protected static string $baseFile;
 
     public function setUp(): void
     {
         parent::setUp();
 
-        self::$localDriverFilePath ??= __DIR__ . '/../storage/documentation.json';
+        $documentationDirectory = config('auto-doc.drivers.local.directory');
+
+        self::$baseFile ??= $documentationDirectory.'/documentation.json';
         self::$documentation ??= $this->getJsonFixture('tmp_data');
 
-        file_put_contents(self::$localDriverFilePath, json_encode(self::$documentation));
+        if (!is_dir(storage_path($documentationDirectory))) {
```

Comment:
I believe we can force set documentation directory as empty for tests, to prevent directory creation on each test run, and implement one test case with the filled directory for test case when documentation should be saved to the configured directory